### PR TITLE
Added Journey Service

### PR DESCRIPTION
**Pull request by ronin1972**

### CODE REVIEW

File: journeys/journeys-service.yaml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      operationId: create-journey
+      requestBody:
+        content:
+          application/json::
+            schema:
+              $ref: '#/components/schemas/Journey'
+
+      responses:
+        '201':
+            description: TeamID for the newly created Team
```
Comment (tommyb82) [inline journeys/journeys-service.yaml:33]: Description incorrect

File: journeys/journeys-service.yaml [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+      parameters:
+        - $ref: '#/components/parameters/JourneyParam'
+      responses:
+        200:
+          description: Journey State
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JourneyState'
+        '400':
```
Comment (tommyb82) [inline journeys/journeys-service.yaml:67]: Can we limit the error responses to only those that can / will be returned by each endpoint?  There are a number here that will never be returned, could confuse the test team.

File: journeys/journeys-service.yaml [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+              $ref: '#/components/schemas/Journey'
+
+      responses:
+        '201':
+            description: TeamID for the newly created Team
+            content:
+                application/json:
+                    schema:
+                        $ref: '#/components/schemas/JourneyId'
+        '401':
```
Comment (tommyb82) [inline journeys/journeys-service.yaml:38]: May want a `409 Conflict` or better still `422 Unprocessable Entity` for handling POST requests with an existing `journey-id`

**Review by tommyb82 — COMMENTED** [review]
Few minor comments 👍

File: journeys/journeys-service.yaml (hunk 1 above)
Comment (ronin1972) [reply journeys/journeys-service.yaml]: Updated

File: journeys/journeys-service.yaml (hunk 2 above)
Comment (ronin1972) [reply journeys/journeys-service.yaml:65]: Removed where I think, hope its right

File: journeys/journeys-service.yaml (hunk 3 above)
Comment (ronin1972) [reply journeys/journeys-service.yaml:38]: Added 422

**Review by tommyb82 — APPROVED** [review]
Looks good - can update as we progress through implementation if further complexities surface..

**Pull request merged**